packages/applications-service-api/src/controllers/representations.js

```diff
@@ -1,23 +1,37 @@
 const { StatusCodes } = require('http-status-codes');
+const { unslugify } = require('unslugify');
 
 const logger = require('../lib/logger');
 const config = require('../lib/config');
 
-const { getRepresentationsForApplication } = require('../services/representation.service');
+const {
+  getRepresentationsForApplication,
+  getFilters,
+} = require('../services/representation.service');
 
 const ApiError = require('../error/apiError');
 
 module.exports = {
   async getRepresentationsForApplication(req, res) {
-    const { applicationId, page } = req.query;
+    const { applicationId, page, searchTerm, type } = req.query;
+
+    const slugified = type && !(type instanceof Array) ? [type] : type;
```

Comment:
The term "slugified" refers to a type of string casing where spaces are replaced with hyphens so they can be better used in URLs.  It doesn't work well here.  To be honest I think we should always treat the `type` filter as an array like so:
```
const types = type instanceof Array ? [...type] : [type];
```